### PR TITLE
Add handling of NOTRACK

### PR DESCRIPTION
**Pull request by junghee**
It is handled by patching mnemonics as other prefixes are done, such as lock or rep.

### CODE REVIEW

File: arch/X86/X86Mapping.c
Comment (adamjseitz) [inline arch/X86/X86Mapping.c]: This line has spaces, but everything else has tabs.

**Review by adamjseitz — CHANGES_REQUESTED** [review]
Just a formatting change, and a few questions:

* Can we add a test case in `suite/cstest/issues.cs`? As an example, I've added some in my previous PRs: https://github.com/capstone-engine/capstone/pull/1877
* Do we have any reason to believe the maintainers won't merge this upstream? It would be better to stay in sync if we can.
* Do we know why tests appear to be failing in GitHub Actions?

File: arch/X86/X86Mapping.c
Comment (junghee) [reply arch/X86/X86Mapping.c]: I just submitted a PR to the main repo: https://github.com/capstone-engine/capstone/pull/2002.

Also, added some tests in issues.cs.

**Review by adamjseitz — APPROVED** [review]
Looks good to me.

**Comment by aeflores** [comment]
This was merged into the official capstone

**Pull request closed without merging**
